### DISCUSSION TITLE
TS18028 on [@types/node]

### DISCUSSION THREAD

**Discussion by ambition-consulting** [DefinitelyTyped/DefinitelyTyped#71046] [Issues with a @types package]
Our build pipeline, that was working for ages with tsify and browserify, has suddenly started producing this funny error

`/builds/software/whatever/@types/node/events.d.ts(144,18): Error TS18028: Private identifiers are only available when targeting ECMAScript 2015 and higher.`

Reverting to `@types/node": "22.8.4"` in devDependencies fixed this.

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment]
Fixed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71054
